rpc/src/rpc.rs

```diff
@@ -119,6 +114,18 @@ use {
     },
 };
 
+#[cfg(test)]
```

Comment:
Not sure where to place this, the best place I've found

Reply:
Seems reasonable